javascript/javascript2/week3/fastestpresser.js

```diff
@@ -0,0 +1,79 @@
+let timerValue = document.getElementById("set-timer").value;
+let setTimer = document.querySelector("#set-timer");
+
+let startBtn = document.querySelector("#start-btn")
+
+function fastestPresser() {
+
+    if (document.getElementById("set-timer").value === "") {
+
+        notetext.innerHTML = "please enter valid time to play!";
+    }
+    else if (!isNaN(document.getElementById("set-timer").value)) {
+        notetext.innerHTML = "The game starts now!";
+    }
+
+}
+
+
+let counterl = 1;
```

Comment:
Since I use keypress if counterl and counters are assigned to 0 the first count of a press would be zero, instead of one.

Reply:
Why aren't you first make counterL and counterS ++ and then print the string? Or instead of counter++, you can use ++counter ( which first ++ the counter and then print it), I think both are better way rather than having the counter starts with 1 since it's going to be confusing later on.